pkg/webui/account.js

```diff
@@ -15,40 +15,35 @@
 import React from 'react'
 import DOM from 'react-dom'
 import { Provider } from 'react-redux'
-import { createBrowserHistory } from 'history'
 import * as Sentry from '@sentry/browser'
 
 import sentryConfig from '@ttn-lw/constants/sentry'
+import store, { history } from '@account/store'
 
-import WithLocale from './lib/components/with-locale'
-import { EnvProvider } from './lib/components/env'
-import Init from './lib/components/init'
-import env from './lib/env'
-import { selectApplicationRootPath } from './lib/selectors/env'
-import './lib/yup'
-import createStore from './account/store'
+import WithLocale from '@ttn-lw/lib/components/with-locale'
+import Init from '@ttn-lw/lib/components/init'
+
+import { selectSentryDsnConfig } from '@ttn-lw/lib/selectors/env'
+
+import '@ttn-lw/lib/yup'
 
-const appRoot = selectApplicationRootPath()
-const history = createBrowserHistory({ basename: `${appRoot}/` })
 // Initialize sentry before creating store
-if (env.sentryDsn) Sentry.init(sentryConfig)
-const store = createStore(history)
-const rootElement = document.getElementById('app')
+if (selectSentryDsnConfig()) {
+  Sentry.init(sentryConfig)
+}
 
 const render = () => {
   const App = require('./account/views/app').default
 
   DOM.render(
-    <EnvProvider env={env}>
```

Comment:
We still have base components that rely on `withEnv` hoc, e.g. `<BaseAnchorLink />` and `<DocLink />` that can be used in the account app. We should keep the `<EnvProvider />` as long as it is used in `@ttn-lw/conponents/`.

Reply:
Searching in `pkg/webui/components` reveals that `<BaseAnchorLink />` and `<DocLink />` seem to be the only one's that still use `withEnv`. I'll add a commit to replace this with env selectors.